### DISCUSSION TITLE
Client TESTCLIENT sent topic with invalid UTF-8, disconnecting.

### DISCUSSION THREAD

**Discussion by Mike0675** [cesanta/mongoose#2439] [Ask questions, get answers] [answered]
Hi, I now get this Error from Mosquitto on my Server:  

Client TESTCLIENT sent topic with invalid UTF-8, disconnecting.

Where can i define the encoding of mongoose output?

Or is it necessary to change the encoding of the hole project?

Which encoding is used best, then?

Regards

**Comment by scaprile** [comment, marked as answer]
Sorry, I don't know what you are talking about, Mongoose is C code, there is no encoding in C, only bytes, octets, and the topic is user (developer) defined. Besides that, TESTCLIENT is also something that does not belong to Mongoose.
